numbergame.py

```diff
@@ -23,12 +24,26 @@ def clearScreen(): #clears screen
     print("The rules of this game are this. \n\t Pick the lower and upper values of the range that the computer will pick the value in \n\t Guess the number, and it will tell you whether you arr too high, or too low. \n\t You want to guess the number in as little tries as possible. \n")
     input("Press ENTER to continue -->")
 
+LOLZ = None
+#LOLZ = True # Uncomment this line to add impossible mode...
+
 #finds out the range and then the number
-lowerNumber = int(input("\n\nWhat is the lower value for the range --> "))
-upperNumber = int(input("What is the upper value for the range --> "))
-number = random.randint(lowerNumber, upperNumber) #picks the number
+lowerNumber = input("\n\nWhat is the lower value for the range --> ")
+upperNumber = input("What is the upper value for the range --> ")
+if LOLZ:
+    lowerNumber = float(lowerNumber)
+    upperNumber = float(upperNumber)
+else:
+    lowerNumber = int(lowerNumber)
+    upperNumber = int(upperNumber)
+if LOLZ:
+    number = random.uniform(lowerNumber, upperNumber)
+else:
+    number = random.randint(lowerNumber, upperNumber) #picks the number
 
 def score(integer): #This is how it calculates which message to give
+    if LOLZ:
+        print("you're a cheater but whatever")

```

Comment:
Why does it say he is a cheater

Reply:
because if they got it right they probably cheated :P

Reply:
change it if you want